ci/README.md

```diff
@@ -0,0 +1,48 @@
+# Tekton CI Pipeline
+
+This directory contains the configuration for a tekton pipeline that tests
+the installed `route-monitor-operator` version in the logged in cluster.
+
+## Installation
+
+A `route-monitor-operator` installation has to be present.
+The rest of the installation is done by running the following command several times:
```

Comment:
Why does the command have to be run several times?

Reply:
Because some of the yamls depend on each other. First, the pipeline operator needs to be installed, then the service account. Then the tekton CRs can be created etc... Sure I could make an own script for that, but it shouldn't matter to much. As running `oc apply -f .` is idempotent